### PR TITLE
Add version to maven plugins

### PR DESCRIPTION
**Pull request by Chusca**
Add version to all the plugins without one.
Plugins changed:
- [Spring boot maven plugin](https://mvnrepository.com/artifact/org.springframework.boot/spring-boot-maven-plugin)

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -53,6 +53,7 @@
             <plugin>
                 <groupId>org.springframework.boot</groupId>
                 <artifactId>spring-boot-maven-plugin</artifactId>
+                <version>2.2.0.RELEASE</version>
```
Comment (PabloGalegoC) [inline pom.xml]: Specifying plugin versions when using Spring is recommended at the parent pom level.

Comment (Chusca) [reply pom.xml]: Solved https://github.com/Chusca/playground-microservices/pull/2/commits/d070f46f7ce4e377abb39f2948d16df49ff26cf9

File: pom.xml [hunk 2]
```diff
@@ -4,6 +4,10 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
+    <prerequisites>
+        <maven>3.6.0</maven>
+    </prerequisites>
+
```
Comment (PabloGalegoC) [inline pom.xml]: Prerequisites tag is not recomended for Maven 3 projects ([link](https://maven.apache.org/pom.html#Prerequisites)).
Old versions of the 'maven versions plugin' may suggest using it anyway.
Better use the 'maven enforcer plugin' and check versions with the latest 'maven versions plugin' ([link](https://github.com/PabloGalegoC/spring-boot-maven-angular-starter/pull/3#issuecomment-553479810))

Comment (Chusca) [reply pom.xml]: Has been changed here https://github.com/Chusca/playground-microservices/pull/2/commits/f4fa0804c1836c2e0e78515b4a654e47d8863c93

**Pull request merged**